app/classifier/normalizer.py

```diff
@@ -76,11 +77,12 @@ def scale_point(point,minX,minY,maxX,maxY,origin):
             return moved_point[0]/drawn_scale*SCALE,moved_point[1]/drawn_scale*SCALE
     return 0,0
 
-def create_normalized_list_of_points(center_of_mass, minX, minY, maxX, maxY, curve_length, signal_list): #list of signals from evtest
+def create_normalized_list_of_points(center_of_mass, minX, minY, maxX, maxY, curve_length, signal_list, colors):
```

Comment:
`pylint` will certainly complain about this function because it has over 5 arguments. You might consider packing all the min max values into a `dict`.